src/App.ts

```diff
@@ -1,4 +1,12 @@
 import { Server } from './Server';
+import { loadConfig } from './utils/ConfigLoader';
+
+/**
+ * Load environment config.
+ */
+const config = loadConfig();
+if (config.error)
```

Comment:
I suggest we always use brackets to avoid potential problems in the future.

Reply:
Also from what I noticed it throw an error when file does not exist which is 👍 . I did not notice same error when file content wasn't in key-value pair format but that is fine.

An example of error:

```
Error: ENOENT: no such file or directory, open '/Users/ddunajski/Projects/lfp-appeals-frontend/build/utils/../../.env.fake'
    at Object.openSync (fs.js:457:3)
    at Object.readFileSync (fs.js:359:35)
    at Object.config (/Users/ddunajski/Projects/lfp-appeals-frontend/node_modules/dotenv/lib/main.js:96:29)
    at Object.exports.loadConfig (/Users/ddunajski/Projects/lfp-appeals-frontend/build/utils/ConfigLoader.js:8:23)
    at Object.<anonymous> (/Users/ddunajski/Projects/lfp-appeals-frontend/build/App.js:5:31)
    at Module._compile (internal/modules/cjs/loader.js:1151:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1171:10)
    at Module.load (internal/modules/cjs/loader.js:1000:32)
    at Function.Module._load (internal/modules/cjs/loader.js:899:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:71:12) {
  errno: -2,
  syscall: 'open',
  code: 'ENOENT',
  path: '/Users/ddunajski/Projects/lfp-appeals-frontend/build/utils/../../.env.fake'
}
```

It looks readable to me so 👍